internal/cache/file/downloader/job.go

```diff
@@ -60,7 +61,6 @@ type Job struct {
 	fileSpec             data.FileSpec
 	// Specifies whether the CRC check needs to be done after the file is
```

Comment:
remove the comment of field being removed.

Reply:
done